packages/assert/README.md

```diff
@@ -3,8 +3,38 @@ assert
 
 Standard type and schema assertions to be used across all 0x projects and packages
 
-## Install
+## Installation
 
 ```bash
-npm install @0xproject/assert --save
+yarn add @0xproject/assert
 ```
+
+## Usage
+
+TODO - Custom documentation page is in progress.
+
+## Contributing
+
+We strongly encourage our community members to help us make improvements and to determine the future direction of the protocol. To report bugs within this package, please create an issue in this repository.
+
+[CONTRIBUTING.md](../../CONTRIBUTING.md)
+
+## Install Dependencies
```

Comment:
Hmm maybe we can leave this out because it belongs in a top level, build-from-source set of instructions, same for all the rest of them

Reply:
Do we need to include install dependencies for any of our packages? Same for all the rest of them

Reply:
I figure it might not be completely obvious to check the top level README if someone wants to only use a single subpackage, just nice to have everything in one place even if there is some duplication.